demos/serverless/deploy.js

```diff
@@ -70,8 +51,7 @@ function usage() {
   console.log(`  -p, --service-principal              Service principal for media pipelines related resources, default is '${chimeMediaPipelinesServicePrincipal}'`)
   console.log(`  -t, --enable-termination-protection  Enable termination protection for the Cloudformation stack, default is false`);
   console.log(`  -l, --disable-printing-logs          Disable printing logs`);
-  console.log(`  -o, --capture-output-prefix          Prefix for S3 bucket name`);
-  console.log(`  -i, --opt-in-regions                 Comma separated list of additional opt-in regions to enable for media capture`);
```

Comment:
This will break the [deploy-canary-demo](https://github.com/aws/amazon-chime-sdk-js/blob/main/script/deploy-canary-demo) script as discussed offline.